src/Core/Configurations/IConfigValidator.cs

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace Azure.DataApiBuilder.Service.Configurations
+namespace Azure.DataApiBuilder.Core.Configurations
 {
 
```

Comment:
nit: `src/Core/Configurations/PermissionOperation.cs` seems to be an empty file left over after merge

Reply:
just removed.